exalearn-api/ExaLearn.Dal/Entities/AssignTest.cs

```diff
@@ -18,5 +19,7 @@ public class AssignTest
         public DateTime ExpirationDate { get; set; }
 
         public LevelType LevelType { get; set; }
+
+        public StatusType Status { get; set; } = StatusType.Active;
```

Comment:
PassedTest has status of user test, why added it to assign test? if the user will pass the test himself without assign test, how you will get status?